salz-core/src/lib/Database.hs

```diff
@@ -54,7 +54,7 @@ getSnapshot cs turn = do
     readMap cl = Map.M $ map readCell $  M.catMaybes $ map liftList cl
 
     readCell :: (Int, Int, Int) -> (Map.Coord, Int)
-    readCell (x, y, pid) = (Map.C (toEnum x) (toEnum y), pid)
```

Comment:
ah the good old days of one letter variable names..

Reply:
I just saw from this that there’s also a `Map.M` 😅

Reply:
Ahh shoot I forgot to change the Map.M constructor.